Views/MenuWidget.cshtml

```diff
@@ -32,7 +32,7 @@
         </button>
         <div id="@contentIdName" class="collapse navbar-collapse @contentElementName">
             <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-                @foreach (var menuItem in viewModel.MenuItems)
+                @foreach (var menuItem in viewModel.MenuItems.OrderBy(item => item.Position))
                 {
                     @if (menuItem.Items.Any())

```

Comment:
Might as well replace the `if` with `viewModel.MenuItems.Where()`. If this was a .cs file the code analysis would demand it anyway.

Reply:
I had to write some generic test code to understand this, and the analyzer does demand it, however not with an `else` present like in this case.